Modules/Core/Common/wrapping/itkPointSet.wrap

```diff
@@ -28,3 +28,12 @@ if(has_d_3)
   itk_wrap_template("VF33" "itk::Vector< float, 3 >, 3")
 endif()
 itk_end_wrap_class()
+
+itk_wrap_class("itk::PointSetBase" POINTER)
```

Comment:
Shouldn't this be at the top of this file, before `PointSet`?

Reply:
Thanks, @dzenanz I just moved the `itk_wrap_class` statement upward, as you suggested, and placed another one in "itkQuadEdgeMeshBase.wrap", please check: [force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/353ef12ef7e736b20b85191ab0c73c0fb283a46d..c0a21a187284906e68af346ad076ebb7945e1a07)

Reply:
For the record, the answer is here: https://github.com/InsightSoftwareConsortium/ITK/pull/4867#issuecomment-2393606971 😺 

Which resolves the conversation, I guess  😇